nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -41,6 +41,14 @@ let
     in
       "${if pkgs.stdenv.buildPlatform == pkgs.stdenv.hostPlatform then Caddyfile-formatted else Caddyfile}/Caddyfile";
 
+  adminDisabled = lib.fileContents (pkgs.runCommand "caddy-config-adapted" {} ''
```

Comment:
That whole check has been added following the review and conversation in https://github.com/NixOS/nixpkgs/pull/217536#discussion_r1136776641

If we move this to `ExecStartPre`, we might as well just remove it entirely again.
The whole point of that check was being able to assert it at eval time.

I personally don't see all that much value in keeping it :thinking:

Reply:
Yeah, moving this assertion to runtime does not add any value since caddy already [fails](https://github.com/NixOS/nixpkgs/pull/217536#discussion_r1114564412) without this assertion.

Reply:
Following this discussion I have removed the check at eval time.